packages/cli/src/commands/deploy/helper.ts

```diff
@@ -77,18 +87,30 @@ export const upsertFieldPlugin: UpsertFieldPluginFunc = async (args) => {
     // update flow
     const mode = skipPrompts ? 'update' : await selectUpsertMode()
     if (mode === 'update') {
+      if (manifest?.options !== undefined) {
```

Comment:
I'm prompting the user confirmation only if a valid manifest file is found and if the user is trying to update the plugin, otherwise, I think it may not be necessary.

I mean, is it necessary to prompt the user during creation time?!

Reply:
can you please make the if statements only to have one level? This will make the readability of the code much better. 

Right now the `if (mode === 'update')` has two nested ifs inside it.

Reply:
I think when a new field plugin is created, the prompt is not necessary.